scrapers/TheNude.yml

```diff
@@ -35,13 +36,13 @@ xPathScrapers:
         postProcess:
           - replace:
               - regex: Asian
-                with: "asian"
+                with: "Asian"
               - regex: Caucasian
-                with: "white"
+                with: "Caucasian"
               - regex: Black
-                with: "black"
-              - regex: Latin
-                with: "hispanic"
+                with: "Black"
+              - regex: Latina
+                with: "Latin"
```

Comment:
The regex replace's where meant to keep the same ethnicity as the first performer scraper we had (freeones)
Some of the changes dont even change anything... I would either leave them as they were or remove all the postProcess

Reply:
No problem. I forget I made other changes to this file locally I didn't mean to push.